R/pkg/R/mllib.R

```diff
@@ -1218,11 +1218,11 @@ setMethod("spark.survreg", signature(data = "SparkDataFrame", formula = "formula
 #' @param data A SparkDataFrame for training.
 #' @param features Features column name. Either libSVM-format column or character-format column is
 #'        valid.
-#' @param k Number of topics.
-#' @param maxIter Maximum iterations.
+#' @param k Number of topics, default is 10.
```

Comment:
OK. There are other places in the document that contains default values. Shall I remove them all?